tests/common.go

```diff
@@ -5311,3 +5311,34 @@ func IsEksPxOperator() bool {
 	}
 	return false
 }
+
+// GetSubsetSlice selects a random subset of unique items from the input slice,
+// with the given length. It returns a new slice with the selected items in random order.
+// If length is zero or negative or greater than the length of the input slice, it also returns an error.
+//
+// Parameters:
+// - items: a slice of any type to select from.
+// - length: the number of items to select from the input slice.
+//
+// Returns:
+// - a new slice of type T with the selected items in random order.
+// - an error if the length parameter is zero or negative, or if it is greater than the length of the input slice.
+func GetSubsetSlice[T any](items []T, length int) ([]T, error) {
```

Comment:
It should be `GetSubsetOfSlice`

Reply:
Renamed "GetSubsetSlice" to "GetSubsetOfSlice".